velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -319,11 +327,16 @@ void AggregationFuzzer::go() {
 
       std::vector<TypePtr> types;
       std::vector<std::string> names;
-
       auto groupingKeys = generateKeys("g", names, types);
+      generateKeys("c", names, types);
       auto input = generateInputData(names, types, std::nullopt);
 
       verifyAggregation(groupingKeys, {}, {}, input, false, {});
+      verifyDistinctAggregation(
```

Comment:
Still seeing hard-coded array_agg and count.

We should add this logic similar to sorted aggs, e.g. in the 'else' branch toss a coin and in 10 or 20% of cases test distinct agg.